src/components/block/block.tsx

```diff
@@ -99,9 +99,14 @@ export class Block implements ConditionalSlotComponent, InteractiveComponent {
 
   /**
    * Block summary.
+   *
+   * @deprecated use description instead
```

Comment:
For consistency, could we change the `@deprecated` line to?:
**Use "description" for the component description text.**